openshift-knative-operator/pkg/monitoring/monitoring_eventing_test.go

```diff
@@ -19,7 +19,7 @@ func TestLoadPlatformEventingMonitoringManifests(t *testing.T) {
 		t.Errorf("Got %d, want %d", len(manifests), 1)
 	}
 	resources := manifests[0].Resources()
-	if len(resources) != 28 {
+	if len(resources) != 25 {
```

Comment:
@skonto can we document why that's actually 28? (and now, after removing 25)?
I have a bit problems with these static numbers... could be a `const` with a better name, and that could have some comments on why

Reply:
Ideally this should be calculated dynamically instead of hardcoding as a number (regardless whether its a constant or not).

Reply:
I will create a PR to make it more dynamic. In general this depends on the number of Eventing deployments as we create monitoring resources per deployment.